assets/js/base/utils/render-frontend.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * External dependencies
```

Comment:
These utils were moved form assets/js/utils since they are shared and used throughout.